### PR TITLE
add: table-datatable-jquery

### PR DESCRIPTION
**Pull request by mrzf833**

### CODE REVIEW

**Comment by zuramai** [comment]
Can you fix the conflicts?

**Comment by mrzf833** [comment]
yeah, conflict is fixed

**Comment by zuramai** [comment]
Can you combine the Datatable menu so it would be like this?
- Datatables
      - Datatable
      - Datatable (jQuery)

File: src/sidebar-items.json [hunk 1]
```diff
@@ -198,6 +198,11 @@
     "url": "table-datatable.html",
     "icon": "file-earmark-spreadsheet-fill"
   },
+  {
+    "name": "Datatable Jquery",
+    "url": "table-datatable-jquery.html",
+    "icon": "file-earmark-spreadsheet-fill"
+  },
```
Comment (zuramai) [inline src/sidebar-items.json:205]: Change this to be a submenu

**Review by zuramai — CHANGES_REQUESTED** [review]
Change the Datatable jQuery menu to submenu

File: src/sidebar-items.json (hunk 1 above)
Comment (mrzf833) [reply src/sidebar-items.json]: finished combine menu datatable

**Review by zuramai — APPROVED** [review]

**Comment by zuramai** [comment]
Great pull request! Thanks

**Pull request merged**